src/front-end/src/components/MenteeForm.js

```diff
@@ -0,0 +1,109 @@
+        </Form.Item>
+        <Form.Item
+          label="Email"
+          name="emailInput"
+          // must have an input:
+          rules={[{ required: true, message: 'Please input something' }]}
+        >
+          <Input placeholder="please enter your email" />
+        </Form.Item>
+        <Form.Item
+          label={
+            <span>
+              Timeframe Seeking Mentorship&nbsp;
+              <Tooltip title="Please pick an estimated duration for how long you would like to use the platform">
+                <QuestionCircleOutlined />
+              </Tooltip>
+            </span>
+          }
+          name="selectedStartDate"
+          rules={[{ required: true, message: 'This field is required' }]}
+        >
+          <RangePicker picker="month" />
+        </Form.Item>
+        <Form.Item
+          label="Please Select Your Time-zone"
+          name="timeZone"
+          // must have an input:
+          rules={[{ required: true, message: 'This field is required' }]}
+        >
+          <TimezonePicker
+            // time zones:
+            absolute={false}
+            defaultValue="America/Los_Angeles"
+            placeholder="Select timezone..."
+            onChange={this.handleChange}
+          />
+        </Form.Item>
+        <p> Please enter your reoccuring availability:</p>
+        <Form.Item label="Monday" name="availabilityMonday">
+          <Select
+            // so that multiple options can be selected:
+            mode="multiple"
+            style={{ width: '100%' }}
+            placeholder="select available time"
+            optionLabelProp="label"
+          >
+            <Option value="9-10AM" label="9-10AM" />
+            <Option value="10-11AM" label="10-11AM" />
+            <Option value="11AM-12PM" label="11AM-12AM" />
+            <Option value="12-1PM" label="12-1PM" />
+            <Option value="1-2PM" label="1-2PM" />
+          </Select>
+        </Form.Item>
+        <Form.Item label="Tuesday" name="availabilityTuesday">
+          <Select
+            // so that multiple options can be selected:
+            mode="multiple"
+            style={{ width: '100%' }}
+            placeholder="select available time"
+            optionLabelProp="label"
+          >
+            <Option value="9-10AM" label="9-10AM" />
+            <Option value="10-11AM" label="10-11AM" />
+            <Option value="11AM-12PM" label="11AM-12PM" />
+            <Option value="12-1PM" label="12-1PM" />
+            <Option value="1-2PM" label="1-2PM" />
+          </Select>
+        </Form.Item>
+        <Button type="primary" htmlType="submit">
+          Submit
+        </Button>
+        <p>{/* used to space buttons */}</p>
+        <Button className ="BackButton-InMentee"type="primary" htmlType="button">
+          Back
+        </Button>
+        <Button className ="NextButton-InMentee"type="primary" htmlType="button">
+          Next
+        </Button>

```

Comment:
"Next" and "Back" buttons are no longer needed since the form can be all on one page, rather than in subsections.

Reply:
done